docs/en/observability/synthetics-feature-roles.asciidoc

```diff
@@ -0,0 +1,124 @@
+// * **{kib} space privileges**: Grant users write or read access to features and apps within {kib}.
+
+[discrete]
+[[synthetics-setup-privileges]]
+= Setup role
+
+Administrators who set up Synthetics typically need to enable Monitor Management.
+Enabling Monitor Management is an admin-level task that requires extra
+privileges. As a best practice, grant the setup role to administrators only, and
+use a more restrictive role for event publishing.
+
+Create a *setup role*, called something like `synthetics_setup`.
+
+. Start with the `editor` {ref}/built-in-roles.html[built-in role].
+This role grants full access to all features in {kib} (including the {observability} solution)
+and read-only access to data indices.
+. Grant the role the following additional privileges, which are required to enable Monitor Management:
++
+[options="header"]
+|====
+| Type | Privilege | Purpose
+
+| {ref}/security-privileges.html#privileges-list-cluster[Cluster]
+| `monitor`
+| Allows the user to retrieve cluster details.
+
+| {ref}/security-privileges.html#privileges-list-cluster[Cluster]
+| `read_ilm`
+| Allows the user access to all read-only index lifecycle management operations.
+
+| {ref}/security-privileges.html#privileges-list-cluster[Cluster]
+| `read_pipeline`
+| Gives the user read-only access to the ingest pipline.
+
+| {ref}/security-privileges.html#privileges-list-indices[Index]
+| `synthetics-*`: `view_index_metadata`
+| Gives the user read-only access to index and data stream metadata.
+
+| {ref}/security-privileges.html#privileges-list-indices[Index]
+| `synthetics-*`: `create_doc`
+| Allows the user to index documents.
+
+| {ref}/security-privileges.html#privileges-list-indices[Index]
+| `synthetics-*`: `auto_configure`
+| Permits auto-creation of indices and data streams.
+
+|====
+
+[discrete]
+[[synthetics-write-privileges]]
+=  Writer role
+
+IMPORTANT: To minimize the privileges required by the writer role, use the
+<<synthetics-setup-privileges,setup role>> to enable Monitor Management.
+This section assumes another user has already enabled Monitor Management.
+
+. Start with the `editor` {ref}/built-in-roles.html[built-in role].
+This role grants full access to all features in {kib} (including the {observability} solution)
+and read-only access to data indices.
+
+. _If the user should have permission to create, modify, and delete project monitors_,
+they will need an API key that can be used to `push` monitors.
+To create API keys, the user will also need _at least one_ of the following privileges in addition to the
+privileges included in the `editor` built-in role:
++
+[options="header"]
+|====
+|Type | Privilege | Purpose
+
+| {ref}/security-privileges.html#privileges-list-cluster[Cluster]
+| `manage_own_api_key`
+| TBD
+
+| {ref}/security-privileges.html#privileges-list-cluster[Cluster]
+| `manage_security`
+| TBD
+
+| {ref}/security-privileges.html#privileges-list-cluster[Cluster]
+| `manage_api_key`
+| TBD
```

Comment:
Did you need feedback from me on the TBD, or will this be more of a general description of the permission?

Reply:
Ah! Yes. Do you think it's helpful to have a brief description for these? I can attempt a generic description based on the Elasticsearch docs if you don't have suggestions!

Reply:
See https://github.com/elastic/observability-docs/pull/2784#pullrequestreview-1357077886 below.